Windows 365 Gateway IP Lookup/W365 Gateway IP Address Retrieval.ps1

```diff
@@ -0,0 +1,151 @@
+﻿<#
+.COPYRIGHT
+Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+See LICENSE in the project root for license information.
+#>
+
+#v1.0
+
+#output CSV file
+$CSVFile = "c:\temp\W365-Gateways.CSV"
+
+#modules required for this script
+$modules = @("az.accounts",
+    "az.network"
+)
+
+#function to ensure modules are installed and current
+function invoke-modulecheck {
+    foreach ($module in $modules) {
+
+        write-host "Checking module - " $module
+        $graphcurrent = (get-installedmodule -name $module -ErrorAction SilentlyContinue)
+        if ($graphcurrent -eq $null) {
+            write-output "Module is not installed. Installing..." | out-host
+            try {
+                Install-Module -name $module -Force -ErrorAction Stop 
+                Import-Module -name $module -force -ErrorAction Stop 
```

Comment:
From the PowerShell document, I noticed that "-force" should be "-Force".

Reply:
Changed.